load_balancing.go

```diff
@@ -48,6 +48,9 @@ type LoadBalancerMonitor struct {
 	Interval      int                 `json:"interval"`
 	ExpectedBody  string              `json:"expected_body"`
 	ExpectedCodes string              `json:"expected_codes"`
+
+	FollowRedirects bool `json:"follow_redirects"`
```

Comment:
Any reason why we aren't grouping these with the remainder of the fields?

Reply:
Nope, i fixed it